### DISCUSSION TITLE
how to cleanup stream state(delete all topics created by streams)

### DISCUSSION THREAD

**Discussion by Ali94k** [LGouellec/streamiz#126] [Q&A] [answered]
Hi,

I am running stream test against Kafka broker on VM. In order to make test results consistent, I want  to start from fresh state every time. How I can cleanup up all footprints of previous tests?

Thanks beforehand!

**Comment by LGouellec** [comment, marked as answer]
Hi @Ali94k,

For now, we haven’t application reset tool like Kafka streams Java.
You can create a script which list all topics prefixed by your application ID and delete them.

Be careful to remove local state store too.

regards

**Reply by Ali94k** [reply]
Dear @LGouellec ,

Thank you for feedback.
